src/CoreWCF.Primitives/src/CoreWCF/Channels/EncoderHelpers.cs

```diff
@@ -6,14 +6,14 @@
 
 namespace CoreWCF.Channels
 {
-    internal static class EncoderHelpers
+    public static class EncoderHelpers
     {
```

Comment:
Keep internal and copy into WebHttp

Reply:
This was addressed in 65884a453baec1859de29ee4c02d571181a5e78a.